src/cljam/util.clj

```diff
@@ -149,8 +149,30 @@
                                  #"(?i)\.(bz2|bzip2)$" :bzip2
                                  nil)))
   ([f k]
-   (let [os (io/output-stream f)]
+   (let [os (cio/output-stream f)]
      (if-let [s (get compressor-map k)]
        (-> (CompressorStreamFactory.)
            (.createCompressorOutputStream s os))
        os))))
+
+;; region utils
+;; ---------
+
+(defn divide-region
+  "Divides a region [start end] into several chunks with maximum length 'step'.
+   Returns a lazy sequence of vector."
```

Comment:
Indent of the second line is incorrect.

```clojure
(defn divide-region
  "Divides a region [start end] into several chunks with maximum length 'step'.
  Returns a lazy sequence of vector."
  ...)
```

is correct.

c.f. [clojure-style-guide#align-docstring-lines](https://github.com/bbatsov/clojure-style-guide#align-docstring-lines)